### ISSUE TITLE
[BITV] Fix left sidebar from a11y perspective

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#40954]
Two issues: 
1. Toggle to close the sidebar is not visible on small displays:

![image](https://github.com/nextcloud/server/assets/42591237/de1cb0de-c6f5-494b-a3ea-80cc1f275a0f)

2. Sidebar does not feature focus trap

https://user-images.githubusercontent.com/42591237/274281340-fecd73f7-668c-4de2-8e76-faa2a67673ab.mp4

Also see the discussion in https://github.com/nextcloud/server/issues/36964#issuecomment-1757697143 and below

**Comment by JuliaKirschenheuter** [comment]
Please keep in mind (Photos): once sidebar is closed it is not possible to open it again

![Peek 2023-10-20 10-00](https://github.com/nextcloud/server/assets/6078378/c92c7b42-5c22-4d03-af1e-d2135dae1f1a)

**Comment by szaimen** [comment]
> Please keep in mind (Photos): once sidebar is closed it is not possible to open it again
> 
> ![Peek 2023-10-20 10-00](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif) [ ![Peek 2023-10-20 10-00](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif) ](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif) [ ](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif)

Needs a vue library update then iirc...

**Comment by szaimen** [comment]
> > ![Peek 2023-10-20 10-00](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif)
> >       
> >         [
> >           
> >         
> >             ![Peek 2023-10-20 10-00](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif)
> >           ](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif)
> >         
> >         
> >           
> >             
> >               
> >             
> >             
> >               
> >               
> >             
> >           
> >           [
> >             
> >               
> >             
> >           ](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif)
> >         
> >        [ ![Peek 2023-10-20 10-00](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif) ](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif)
> >       
> >         [
> >           
> >         
> >             ![Peek 2023-10-20 10-00](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif)
> >           ](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif)
> >         
> >         
> >           
> >             
> >               
> >             
> >             
> >               
> >               
> >             
> >           
> >           [
> >             
> >               
> >             
> >           ](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif)
> >         
> >        [ ](https://user-images.githubusercontent.com/6078378/276863574-c92c7b42-5c22-4d03-af1e-d2135dae1f1a.gif)
> 
> Needs a vue library update then iirc...

WIP here: https://github.com/nextcloud/photos/pull/2111
